iothub/device/src/IotHubModuleClient.cs

```diff
@@ -87,14 +87,20 @@ internal IotHubModuleClient(IotHubConnectionCredentials iotHubConnectionCredenti
         /// <returns>A disposable client instance.</returns>
         public static async Task<IotHubModuleClient> CreateFromEnvironmentAsync(IotHubClientOptions options = default)
         {
-            // Make sure client options is initialized.
-            if (options == default)
-            {
-                options = new();
-            }
+            IotHubClientOptions clientOptions = options != null
+                ? new ()
+                {
+                    FileUploadTransportSettings = options.FileUploadTransportSettings,
```

Comment:
I don't think you can do file upload from a module client, so this can be removed.
Right, @timtay-microsoft ?

Reply:
We don't use this when building an HTTP client to invoke methods, right? If not, then by all means remove this line